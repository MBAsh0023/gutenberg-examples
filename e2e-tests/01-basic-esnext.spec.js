/**
 * WordPress dependencies
 */
import {
	getEditedPostContent,
	insertBlock,
	createNewPost,
} from '@wordpress/e2e-test-utils';

/**
 * Internal dependencies
 */
import json from '../01-basic-esnext/block.json';
const { title, name } = json;

// Increase the timeout limit for this test.
jest.setTimeout( 100000 );

it( `${ title } block should be available`, async () => {
	await createNewPost();
	await insertBlock( title );

	// Check if block was inserted
	expect( await page.$( `[data-type="${ name }"]` ) ).not.toBeNull();

	expect( await getEditedPostContent() ).toMatchInlineSnapshot( `
		"<!-- wp:gutenberg-examples/example-01-basic-esnext -->
		<div style=\\"background-color:#900;color:#fff;padding:20px\\" class=\\"wp-block-gutenberg-examples-example-01-basic-esnext\\">Hello World, step 1 (from the frontend).</div>
		<!-- /wp:gutenberg-examples/example-01-basic-esnext -->"
	` );
} );
