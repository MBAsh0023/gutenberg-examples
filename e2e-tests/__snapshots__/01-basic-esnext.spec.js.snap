// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example: Example: Basic (ESNext) block should be available 1`] = `
"<!-- wp:gutenberg-examples/example-01-basic-esnext -->
<div style=\\"background-color:#900;color:#fff;padding:20px\\" class=\\"wp-block-gutenberg-examples-example-01-basic-esnext\\">Hello World, step 1 (from the frontend).</div>
<!-- /wp:gutenberg-examples/example-01-basic-esnext -->"
`;
